openedx/frontend-app-admin-portal
# feat: add removed members count

# Description
Adds the removed learners count to toggle in the Members tab when there are removed members.
https://2u-internal.atlassian.net/browse/ENT-8968

# UI Updates
|figma|before|after|
|------|------|-----|
|![Screenshot 2024-05-22 at 2 11 05 PM](https://github.com/openedx/frontend-app-admin-portal/assets/71999631/e6da70c4-d3e2-4b4b-9d7f-343512d4ebc3)|![Screenshot 2024-05-22 at 2 13 14 PM](https://github.com/openedx/frontend-app-admin-portal/assets/71999631/48efeeed-a175-4f1d-808b-e467e5144a69)|![Screenshot 2024-05-22 at 2 11 59 PM](https://github.com/openedx/frontend-app-admin-portal/assets/71999631/539f91f6-4b00-4297-a0be-b4dc6758f14a)|

# Test plan
1) checkout branch `knguyen2/ent-8968`. in your terminal, run the follow commands:
     - `git fetch`
     - `git checkout knguyen2/ent-8968`
     - `npm run start:stage`
2) use staging link https://localhost.stage.edx.org:1991/exec-ed-2u-integration-qa/admin/learner-credit/92fa52bc-6863-49ff-8a32-1cfefbfbb728/members to verify that the count of removed learners is in the toggle
3) confirm that when toggled is on, the removed learner should appear in the data table
![Screenshot 2024-05-22 at 2 18 25 PM](https://github.com/openedx/frontend-app-admin-portal/assets/71999631/2cd8c9be-de19-4b1f-a392-ec0a10c7d7a8)
4) verify changes match figma mocks
https://www.figma.com/design/H69Met5YL7xu5w5LRliumQ/Groups-MVP?node-id=4289-23202&t=0DKGA6qrlzZz0EyD-0

# For all changes


# Only if submitting a visual change
